librz/util/lib.c

```diff
@@ -199,21 +52,19 @@ RZ_API void rz_lib_free(RzLib *lib) {
 	free(lib);
 }
 
-static bool __lib_dl_check_filename(const char *file) {
+static bool lib_dl_check_filename(const char *file) {
 	return rz_str_endswith(file, "." RZ_LIB_EXT);
 }
 
-RZ_API int rz_lib_run_handler(RzLib *lib, RzLibPlugin *plugin, RzLibStruct *symbol) {
+static bool lib_run_handler(RzLib *lib, RzLibPlugin *plugin, RzLibStruct *symbol) {
 	RzLibHandler *h = plugin->handler;
-	if (h && h->constructor) {
-		RZ_LOG_DEBUG("PLUGIN LOADED %p fcn %p\n", h, h->constructor);
+	if (h->constructor) {
 		return h->constructor(plugin, h->user, symbol->data);
```

Comment:
this does not make sense. how a user can set the `h->user` pointer if you just pass a pointer (i.e. `void *`) instead of the reference to the pointer (i.e. `void**`) ?

Reply:
This is data to be passed to the plugin-type handler. Why do you need to set the user? I'm not understanding what you want to achieve exactly.

Reply:
ignore me.